EDDiscovery/EDDiscovery.csproj

```diff
@@ -1134,8 +1134,10 @@
       </ZipFiles>
       <Zipfiles Include="$(OutDir)x64\SQLite.Interop.dll">
         <Name>x64\SQLite.Interop.dll</Name>
+          <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
Trying to work out how this would make a difference - you're saying to copy the file from the output directory to the output directory.

Reply:
I agree, won't pull that in.

Reply:
I looked up the error I was getting when building, and saw this post.  Gave it a shot. Code built and ran afterwards. 

https://stackoverflow.com/questions/13028069/unable-to-load-dll-sqlite-interop-dll

Reply:
I suspect that modifying the csproj outside visual studio resulted in the project being rebuilt, thus pulling the interop files from the EliteDangerous project, which has:

https://github.com/EDDiscovery/EDDiscovery/blob/a595339420e569e31d18e2aa21fa5e9f9d299818/EliteDangerous/EliteDangerous.csproj#L17-L20